addons/website/static/src/snippets/s_process_steps/options.js

```diff
@@ -0,0 +1,56 @@
+odoo.define('website.s_process_steps_options', function (require) {
+    'use strict';
+    const options = require('web_editor.snippets.options');
+
+    options.registry.ConnectorChoice = options.Class.extend({
+        /**
+         * @override
+         */
+        start: function () {
+            if (!this.$target[0].dataset.currentConnector) {
+                this.$target[0].dataset.currentConnector = "line";
+            }
+            return this._super(...arguments);
```

Comment:
Probably the kind of stuff which should be done in `onBuilt` instead of `start` but in fact better: not needed at all. Just consider that the default is "line".